README.md

```diff
@@ -53,6 +53,7 @@ const Waterwheel = require('waterwheel');
 const waterwheel = new Waterwheel('http://test.dev', {username: 'admin', 'password': '1234'});
 
 // Browser
+import '../../path/to/node_modules/waterwheel/dist/waterwheel.js'
```

Comment:
I think the expectation with the browser version of `waterwheel` would be that the built version would be included via a `script` tag, although I guess this is technically possible.